custom/icds_reports/utils/aggregation_helpers/distributed/agg_awc_daily.py

```diff
@@ -123,6 +124,33 @@ def update_query(self):
             'date': self.date
         }
 
+    def update_launched_query(self):
+        return """
+        CREATE TEMPORARY TABLE "{temp_table}" AS
+            SELECT
+                owner_id,
+                supervisor_id,
+                sum(open_count) AS cases_household,
+                count(*) AS all_cases_household
+            FROM "{household_cases}"
+            WHERE opened_on<= %(end_date)s
+            GROUP BY owner_id, supervisor_id;
+        UPDATE "{tablename}" agg_awc SET
+           cases_household = ut.cases_household,
+           num_launched_states = CASE WHEN ut.all_cases_household>0 THEN 1 ELSE 0 END,
+           num_launched_districts = CASE WHEN ut.all_cases_household>0 THEN 1 ELSE 0 END,
+           num_launched_blocks = CASE WHEN ut.all_cases_household>0 THEN 1 ELSE 0 END,
+           num_launched_supervisors = CASE WHEN ut.all_cases_household>0 THEN 1 ELSE 0 END,
+           num_launched_awcs = CASE WHEN ut.all_cases_household>0 THEN 1 ELSE 0 END
+        FROM (
+            SELECT * FROM "{temp_table}"
+        ) ut
+        WHERE ut.owner_id = agg_awc.awc_id and ut.supervisor_id=agg_awc.supervisor_id;
```

Comment:
I know its implicit here that it will only update aggregation_level=5 rows. But would be good to apply that filter  explicit both for clarity and some performance improvement